prql-compiler/src/test.rs

```diff
@@ -1,8 +1,16 @@
 //! Simple tests for "this PRQL creates this SQL" go here.
-use super::*;
-use crate::parser::parse;
+// use super::*;
+use crate::{parser::parse, sql};
 use insta::{assert_display_snapshot, assert_snapshot};
 
+pub fn compile(prql: &str) -> Result<String, crate::ErrorMessages> {
```

Comment:
FWIW I do find myself asking if the functions are useful for seeing test outputs, surely they're useful as top-level methods in the crate?

Maybe there are defaults for tests that are specific, but the use case of "convert prql to sql with default options" seems fairly general!

Reply:
You're right. `compile` should also take `Option<Options>`.